x/pdv/handler.go

```diff
@@ -9,11 +9,16 @@ import (
 	cerberusapi "github.com/Decentr-net/cerberus/pkg/api"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
 // NewHandler creates an sdk.Handler for all the pdv type messages
-func NewHandler(cerberus cerberusapi.Cerberus, keeper Keeper) sdk.Handler {
+func NewHandler(keeper Keeper) sdk.Handler {
+	pk := secp256k1.PrivKeySecp256k1{}
+
 	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
+		cerberus := cerberusapi.NewClient(keeper.GetCerberusAddr(ctx), pk)
```

Comment:
@mikluke need to check performance

Reply:
lets create client in NewHandler

Reply:
so also we need some mechanism to react on keeper.SetCerberus

maybe we should create a client in the keeper?